coffee-module/coffee-module-redisstream/pom.xml

```diff
@@ -22,7 +22,17 @@
 			<groupId>org.jboss.weld</groupId>
 			<artifactId>weld-api</artifactId>
 			<version>3.0.Final</version>
-		</dependency>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.enterprise</groupId>
+					<artifactId>cdi-api</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>javax.inject</groupId>
+					<artifactId>javax.inject</artifactId>
```

Comment:
ezt sem latom a dependencykben

Reply:
jogos, a cdi-api húzta be tranzitívan, de azt pont egyel feljebb exclude-oltam, így ez elhagyható. kiszedem.